### ISSUE TITLE
Retire this package

### ISSUE THREAD

**Issue by keszybz** [fedora-infra/python-fedora#235]
This package uses `kitchen`. The repo for `kitchen` was archived on August 9th as part of the Fedora infra hackfest during Flock.

Packaging deps in Fedora 40 show two dependencies on `fedora`:
`python3-packit` and `fedpkg`, and both are unnecessary.
PRs to remove those imports and drop the dependency: https://github.com/packit/packit/pull/2381 and TBD.

If there are no other uses, we can retire `fedora` and `kitchen` in F41.

/cc @AdamWill

**Comment by AdamWill** [comment]
Agreed. I'm still working on my fedpkg PR, it keeps...metastasizing. But should be done by EOD.

**Comment by AdamWill** [comment]
fedpkg pr is https://pagure.io/fedpkg/pull-request/560 .
